packages/waku/src/lib/plugins/vite-plugin-rsc-env.ts

```diff
@@ -11,6 +12,12 @@ export function rscEnvPlugin({
   return {
     name: 'rsc-env-plugin',
     config(viteConfig) {
+      dotenv.config({
+        path: ['.env.local', '.env'],
+        processEnv: (globalThis as any).__WAKU_PRIVATE_ENV__,
+        override: true,
+      });
```

Comment:
Hm, I'm not sure how it works, but can we make this explicitly for dev only?

Reply:
For sure!